modules/weko-search-ui/weko_search_ui/utils.py

```diff
@@ -717,6 +711,21 @@ def compare_identifier(item, item_exist):
     return item
 
 
+def make_tsv_by_line(lines):
+    """Make TSV file."""
+    import csv
+    from io import StringIO
```

Comment:
Hi,
Duplicated definition this import with function make_stats_tsv. Plz handle this.